tasks/basecalling/task_dorado_demux.wdl

```diff
@@ -0,0 +1,95 @@
+version 1.0
+
+task dorado_demux {
+  input {
+    Array[File] bam_files
+    String kit_name
+    String fastq_file_name
+    Int cpu = 4 
+    Int memory = 16
+    Int disk_size = 50
```

Comment:
storage is cheap and there's a good chance of running out of disk space if input BAMs and output FASTQ files are large. I'd increase default disk_size to at least 100 GB. Imagine a full GridION run with 5 flowcells worth of data being basecalled in a single workflow. Could easily surpass 50GB